core/capabilities/ccip/ccipevm/addresscodec_test.go

```diff
@@ -24,34 +24,33 @@ func TestAddressStringToBytes(t *testing.T) {
 	require.Equal(t, want, got)
 }
 
+// we allow various sizes since some contracts store the 20-byte address as 32-byte
```

Comment:
Maybe also add a comment in the AddressBytesToString()
 from `core/capabilities/ccip/ccipevm/addresscodec.go`, as for reason we skip the length check